smarts/core/sensors.py

```diff
@@ -862,7 +862,7 @@ def append_waypoint_if_new(self, new_wp):
             or wp_edge in self._mission_planner.route.edges
         )
 
-        if most_recent_wp != new_wp and should_count_wp:
+        if np.linalg.norm(new_wp.pos - most_recent_wp.pos) > 0.5 and should_count_wp:
```

Comment:
Do we have a constant or variable we can use to describe what is going on here instead of 0.5?

Reply:
good point, I'll move it to a variable.